### ISSUE TITLE
Design discovery for the keyboard map move trigger invisibility issue

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#23053]
## Describe the defect
In a previous review we got the comment "Another wishlist item: keyboard-accessible controls for moving the map north/south/east/west."

Currently, the map is movable with arrow keys IF you have the ‘center point’ focused. It’s just hard to tell that it’s an option. There is no visual indicator that there is a center point to focus and no focus indicator when it is in focus. We can’t have invisible focusable items.

We need to determine how we can provide keyboard navigation of the map without using an invisible focus point. Some initial questions:
- Do we need that ‘center’ invisible focus point?
- Is there another ‘focus point’ that could be used to trigger arrow map nav?
- Can we make that center focus visible and clearly interactive?
- Are there standard navigation patterns that users would expect? Any that are so well established that instructions are unneeded? Or common ways to provide instructions?

Secondarily (ticket separately if desired), consider what instructions, if any, are useful. Currently, map instructions are not visible and only available to screen readers.
- How do we inform people that this navigation option exists?
- What do people need to know to navigate successfully? 
- Do we need to provide instructions at all?
- Are visible instructions an option? If not, how will users who need these instructions get them?

#### Current behavior:
- Currently, the map is navigable with arrow keys IF you have the ‘center point’ focused. 
  - There is no visual indicator that there is a center point to focus and no focus indicator when it is in focus.
- Currently the instructions for navigating with arrows are not reliably announced and are visually hidden
  - Map instructions message: “Search areas on the map up to a maximum of 500 miles. Zoom in or out using the zoom in and zoom out buttons. Use a keyboard to navigate up, down, left, and right in the map.”
  - Currently the message is populated when you tab to the map container focus point. Then tab once more to get to the ‘center point’ that actually allows for arrow navigation.
  - The ‘map instructions’ are referenced in the map container div#mapbox-gl-container with aria-describedby. Map instructions also has aria-live=assertive. 
  - I did not hear the description announced when the map received focus in VO/FF. 
  - When the map gets focus, the map instructions div gets content. But I did not hear the live region announce. That’s weird…didn’t see anything out of the ordinary and expected an initial announcement.
  - But I could access it through the VoiceOver prompts for the “more content menu’  
- Map is also draggable. 

## To Reproduce
1. Go to https://www.va.gov/find-locations
2. Tab to the map.
3. Note that on first focusing themap,it gets a yellow focus border around the whole map.
4. Tab once more (you are now focused on the invisible tab stop. Try arrow navigation now!)
5. Note no visible focus on the page.
6. Tab once more and note that the zoom + now has visible focus 
 
## AC / Expected behavior
- [ ] 

## Screenshots

<details>
<summary>

</summary>

</details>
